### ISSUE TITLE
Add `nodejs8.10` to list of supported platforms in CLI error message

### ISSUE THREAD

**Issue by vboctor** [serverless/serverless#4992]
# This is a Bug Report

## Description

For bug reports:
* What went wrong?

An error occurred: XyzLambdaFunction - 1 validation error detected: Value 'nodejs8.1' at 'runtime' failed to satisfy constraint: Member must satisfy enum value set: [java8, nodejs, nodejs4.3, nodejs6.10, python2.7, python3.6, dotnetcore1.0].

* What did you expect should have happened?

I would expect that `nodejs8.10` would be listed in the supported runtimes.

* What was the config you used?

sls v1.27.2 deploying to AWS Lambda

* What stacktrace or error message from your provider did you see?

See above.

For feature proposals:
* What is the use case that should be solved. The more detail you describe this in the easier it is to understand for us.
* If there is additional config how would it look

Similar or dependent issues:

## Additional Data

* ***Serverless Framework Version you're using***: v1.27.2
* ***Operating System***: Mac OS X 10.13.4
* ***Stack Trace***: n/a
* ***Provider Error messages***:

See above.

**Comment by HyperBrain** [comment]
@vboctor Which command did you execute? I'm sure that Node 8.10 works properly (we already have switched at work).
According to the error message you posted above `Value 'nodejs8.1' at ...` the zero seems to be missing.
Can you please double check?

**Comment by vboctor** [comment]
@HyperBrain The use of `nodejs8.10` works.  However, when I used `nodejs8.1` (without the 0) I got an error message that lists the supported platforms and `nodejs8.10` wasn't listed in the supported platforms.  So it is an error in the command line tool error message.

**Comment by HyperBrain** [comment]
@vboctor Thank you for the clarification. I agree that this should be fixed to be consistent 😄

**Comment by rohitshetty** [comment]
@HyperBrain I am new to contributing to serverless, although I have been using serverless for more than a year now for my client projects. I would like to start contributing to serverless, given how amazing this framework is.
This looks like a simple fix and would love to take this up if someone already has not taken this up.

**Comment by HyperBrain** [comment]
Hi @rohitshetty , thanks 🥇 . It is great that you want to contribute. Please feel free to do your first contribution with this issue. If you have any questions during your implementation, just ask 😄 
I'll set it to in progress, so that it is obvious that someone is working on it.

**Comment by rohitshetty** [comment]
Thank you very much for the support and encouragement @HyperBrain !

**Comment by rohitshetty** [comment]
@HyperBrain I tried to recreate the above bug and tried tracing through the code. 
At serverless level we don't validate the lambda runtime this error is rather thrown because of cloudformation stack creation failure for lambda. 
I verified the failed cloudformation stack console output and saw the same error message with nodejs8.10 missing in the list. 
TLDR: This inconsistency is from the AWS end and not from the serverless framework, and there is nothing we can do about this.
Please correct me if I am wrong about serverless not validating the runtime type.

**Comment by rohitshetty** [comment]
[Here is the failed cloudformation console output for reference](https://ibb.co/bsHyrT)

**Comment by HyperBrain** [comment]
@rohitshetty Thanks for the analysis. I think you're right. The only thing I wonder about is, that Serverless actually does call the template verification AWS API to validate the generated template before CF is told to execute the template.
So, if AWS would work properly, the error should even be triggered earlier than now.
However, IMO there's not much we can do but report the issue to AWS (including the different behavior between actual deployment and prior validation).

**Comment by HyperBrain** [comment]
@rohitshetty A report to the AWS forums sounds like a good start and see if there is any feedback from AWS.

**Comment by exoego** [comment]
This seems is fixed in upstream (AWS CloudFormation).

Error message returned from AWS CF now contains `nodejs8.10` and others like `go 1.x`

```
An error occurred: HelloLambdaFunction - Value INVALID at 'runtime' failed to satisfy constraint: Member must satisfy enum value set: [java8, nodejs6.10, nodejs8.10, python2.7, python3.6, dotnetcore1.0, dotnetcore2.0, dotnetcore2.1, go1.x] or be a valid ARN (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: ...).
```

**Comment by exoego** [comment]
Closing this since AWS CloudFormation have already added `nodejs8.10` and other newer runtimes to the error message (see above comment).

Feel free to reopen if `nodejs8.10` is still missing.
